docs/source/use_cases/privacy_protection_people_screens.rst

```diff
@@ -41,24 +39,24 @@ PeekingDuck with the configuration file |pipeline_config|_ as shown:
 How It Works
 ============
 
-There are <number> main components to our solution:
+There are 2 main components to our solution:
 
-#. Human and Computer Screen Segmentation, and
-#. Human and Computer Screen blurring.
+#. Person and Computer Screen Segmentation, and
+#. Person and Computer Screen Blurring.
 
-**1. Human and Computer Screen Segmentation**
+**1. Person and Computer Screen Segmentation**
 
-We use an open source instance segmentation model known as <to-be-inserted-with-link> to obtain the
-masks of persons, computer screens and laptops. The masks are akin to the input frames or images,
-except that it only has a single channel and each pixel on the mask is a binary of either 1 or 0,
-which indicates whether a specific class of thing is present (1) or absent (0) in a particular
-location of the image. For more information on how to adjust the <insert-model-name> node, check
-out the <insert-link-to-model-page>.
+We use an open source instance segmentation model known as `Mask R-CNN <https://arxiv.org/abs/1703.06870>`_
+to obtain the masks of persons, computer screens and laptops. The masks are akin to the input frames
+or images, except that it only has a single channel and each pixel on the mask is a binary of either
+1 or 0, which indicates whether a specific class of thing is present (1) or absent (0) in a
+particular location of the image. For more information on how to adjust the Mask_RCNN node, check
+out the :doc:`Mask_RCNN configurable parameters </nodes/model.mask_rcnn>`.
```

Comment:
Would it be better to change to the following instead, following the zone_counting.rst convention?

```suggestion
particular location of the image. For more information on how to adjust the ``mask_rcnn`` node, check
out its :doc:`configurable parameters </nodes/model.mask_rcnn>`.
```

Reply:
FYI, I had followed the copy from privacy_protection_faces.rst.

In any case, I have implemented your suggested amendment.